src/hooks/useFetchItems.ts

```diff
@@ -621,3 +622,75 @@ export const useGetGroupsUpcomingEpisodes = (parentId: ParentId) => {
         enabled: !!parentId
     });
 };
+
+interface ToggleFavoriteMutationProp {
+    itemId: string;
+    isFavorite: boolean | undefined
+}
+
+const fetchUpdateFavoriteStatus = async (
+    currentApi: JellyfinApiContext,
+    itemId: string,
+    isFavorite: boolean | undefined
+) => {
+    const { api, user } = currentApi;
+    if (api && user?.Id) {
+        if (isFavorite) {
+            const response = await getUserLibraryApi(api).unmarkFavoriteItem({
+                userId: user?.Id,
```

Comment:
This check is redundant

```suggestion
                userId: user.Id,
```